src/gtk/window.cpp

```diff
@@ -1594,6 +1594,12 @@ gtk_window_button_press_callback( GtkWidget* WXUNUSED_IN_GTK3(widget),
                                   GdkEventButton *gdk_event,
                                   wxWindowGTK *win )
 {
+    /* gtk does not set the button1 mask when the event comes from the left button of a mouse.
+      but from some reason, it sets it when the event comes from a touchscreen
+      So simply remove it for consistency
+    */
+    gdk_event->state &= ~GDK_BUTTON1_MASK;
+
```

Comment:
Could we do it before calling this function in that case? It seems like it would be more appropriate there.

Reply:
It really needs to be done here, actually.
My fault is to not have more explained this in the comment, and I will do so, sorry.
Even when -not- using the gesture events, gtk sets that bit when the source device is a touchscreen (at least,
the touchscreen I am using for testing), and there does not seem to be possible detect that in the event information.

Reply:
... and that does not break the behaviour when using a mouse, or a touchscreen that performs a better mouse emulation, like my Thinkpad T480, for instance.

Reply:
I see, thanks for explaining it!

Reply:
your welcome, hopefully the modified comment now helps more.